timesketch/app.py

```diff
@@ -91,6 +91,16 @@ def create_app(config=None):
                          '$ openssl rand -base64 32\n\n')
         sys.exit()
 
+    # Support old style config using Elasticsearch as backend.
+    # TODO: Deprecate the old ELASTIC_* config in 2023.
+    if not app.config.get('OPENSEARCH_HOST'):
+        sys.stderr.write('Deprecated config field found: ELASTIC_HOST\n')
```

Comment:
I know it is not likely, but shouldn't you check if the elastic value still exists at that point? Implying because you found no opensearch host that there should be an elastic_host could be wrong, WDYT?

Reply:
The ELASTIC_HOST one has been there since the start, and if people remove that they won't be able to start the server. I opt to let this be.

Reply:
ACK